### ISSUE TITLE
Error recognition  for KubeProxyConfiguration 

### ISSUE THREAD

**Issue by libratiger** [rikatz/kubepug#334]
for k8s 1.18 - k8s 1.25 version.

```
KubeProxyConfiguration found in kubeproxy.config.k8s.io/v1alpha1
	 ├─ API REMOVED FROM THE CURRENT VERSION AND SHOULD BE MIGRATED IMMEDIATELY!!
```

the `KubeProxyConfiguration` has only v1alpha1 now.

**Comment by rikatz** [comment]
Can you test with https://github.com/rikatz/kubepug/releases/tag/v1.5.0-alpha.0 please? :)

Thanks!

**Comment by rikatz** [comment]
As this API is not a real kubernetes API for now, it should not be reported

https://kubepug.xyz/deprecations/ contains the new deprecated and deleted DBs, I couldn't find it :) 

I'm cutting a release with this new DB soon

**Comment by rikatz** [comment]
Fixed in #419
